src/displayapp/DisplayApp.cpp

```diff
@@ -264,6 +275,10 @@ void DisplayApp::LoadApp(Apps app, DisplayApp::FullRefreshDirections direction)
         this, notificationManager, systemTask.nimble().alertService(), Screens::Notifications::Modes::Preview);
       ReturnApp(Apps::Clock, FullRefreshDirections::Up, TouchEvents::SwipeUp);
       break;
+    case Apps::Timer:
+      currentScreen = std::make_unique<Screens::Timer>(
```

Comment:
I think this here is misconfigured max line width.

Reply:
its just the result of copying a line that was warped and editing it. fixed it now